config/templates/cluster.yaml

```diff
@@ -171,8 +171,10 @@ controller:
 # subnets:
 #   - availabilityZone: us-west-1a
 #     instanceCIDR: "10.0.0.0/24"
+#     subnetId: "subnet-xxxxxxxx" #optional
 #   - availabilityZone: us-west-1b
 #     instanceCIDR: "10.0.1.0/24"
+#     subnetId: "subnet-xxxxxxxx" #optional
```

Comment:
Note: `subnetId` will be renamed to `id` in the future according to https://github.com/coreos/kube-aws/pull/195#issuecomment-272061515
cc @icereval